### DISCUSSION TITLE
Support rootfs sub-command

### DISCUSSION THREAD

**Discussion by bgoareguer** [aquasecurity/trivy-operator#783] [Development]
Trivy-operator already supports the image and filesystem subcommands.

The image subcommand requires to pull the images to scan from the registries (which should not be required since the images are already in the local cache of each K8S node). Using the filesystem subcommand does not require to pull the images but it does not scan the same packages (https://aquasecurity.github.io/trivy/v0.35/docs/vulnerability/detection/language/).

I think it would make sense to add support for the rootfs subcommand so that we can have the same result as the image subcommand but without pulling the images from the registries.

What do you think?

**Comment by chen-keinan** [comment]
@bgoareguer could be a nice addition to our existing scanning capabilities , I think opening an issue for it. could be a start of it
